src/jimgw/samplers/flowmc.py

```
@@ -96,10 +95,14 @@ def __init__(
         self._strategy_order_from_config: list[str] = order
 
     # flowMC expects callables with signature (params, data) -> Float.
-    def _logpdf_flowmc(self, params: Float[Array, " n_dims"], _data: dict) -> Float:  # noqa: F722
+    def _logpdf_flowmc(
+        self, params: Float[Array, " n_dims"], _data: dict
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Expected an expression

(F722)